ingeniamotion/utils.py

```diff
@@ -0,0 +1,24 @@
+from typing import Union
+
+from ingenialink.ethernet.register import EthernetRegister
+from ingenialink.canopen.register import CanopenRegister
+
+
+def map_register_address(register: Union[EthernetRegister, CanopenRegister]) -> int:
```

Comment:
Don't create an utils.py file, it is an antipattern https://www.yanglinzhao.com/posts/utils-antipattern/.
And create a test for this function.